Update My profile (#15)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/interface/profile.md

You can see your account details or, if you don't have one, create a new one for free.

## Create an account

 1. Start by going to the **Home** screen.
 2. In the top-right corner click on **Create account**.

![](/gdevelop5/interface/profile/home not logged.png)

 3. Enter your username, fill your email address, and choose a password.
 4. Then click **Create account**.
 5. To confirm your account creation, you will receive an email with a link to click on. 

![](/gdevelop5/interface/profile/account-create-details.png)

Once you're logged into your account, you can see your profile. You can then use GDevelop specific services like packaging your game for multiple support in one click!

### Customize your profile

You can customise your public profile information directly from the GDevelop interface to tell more about you as a game creator. The details filled will all be displayed on [gd.games](https://gd.games/) the GDevelop's gaming platform and on your GDevelop user page that appears if someone clicks on your name on one of your contribution such as extensions, examples, and assets.

![](/gdevelop5/interface/profile/gdevelop_profile_customize.gif)

> A profile can have some properties:

Reviewer: Profiles have different properties:

- **An username**. Your unique username on GDevelop's websites and services.
- **A biography**. Introduce yourself in a few words, tell us who you are and what you do.

> - **A donation link**. Useful to bind your Patreon, Paypal, Ko-fi, or any URL.

Reviewer: - **A donation link**. Useful for linking your Patreon, Paypal, Ko-fi, or any URL.

> - **A donation link**. Useful to bind your Patreon, Paypal, Ko-fi, or any URL.

Author: ```suggestion
- **A donation link**. Useful for linking your Patreon, Paypal, Ko-fi, or any URL.
```

> - Is your subscription to the **GDevelop Newsletter** is active.

Reviewer: - If your subscription to the **GDevelop Newsletter** is active.

> - Is your subscription to **the analytics of your games** is active.

Reviewer: - If your subscription to **the analytics of your games** is active.

> - Is your subscription to **the analytics of your games** is active.

Author: ```suggestion
- If your subscription to **the analytics of your games** is active.
```

- The current **GDevelop subscription** running on your account.

### Changing your profile picture

> GDevelop does not host and control profile pictures. Instead, it obtains your profile picture from [Gravatar](https://gravatar.com). If you have or create a [Gravatar](https://gravatar.com) account, you can link a new profile picture to the email address you used to register your GDevelop account. This will change automatically your profile picture in GDevelop as well.

Reviewer: GDevelop does not host or control profile pictures. Instead, it obtains your profile picture from [Gravatar](https://gravatar.com). If you have or create a [Gravatar](https://gravatar.com) account, you can link a new profile picture to the email address you used to register your GDevelop account. This will automatically change your profile picture in GDevelop as well.

> GDevelop does not host and control profile pictures. Instead, it obtains your profile picture from [Gravatar](https://gravatar.com). If you have or create a [Gravatar](https://gravatar.com) account, you can link a new profile picture to the email address you used to register your GDevelop account. This will change automatically your profile picture in GDevelop as well.

Author: ```suggestion
GDevelop does not host or control profile pictures. Instead, it obtains your profile picture from [Gravatar](https://gravatar.com). If you have or create a [Gravatar](https://gravatar.com) account, you can link a new profile picture to the email address you used to register your GDevelop account. This will automatically change your profile picture in GDevelop as well.
```

## Subscription

### Get a subscription

> With the free plan the online services are limited. Limited services rely on servers that can be costly.

Reviewer: With the free plan, the online services are limited. These services rely on servers which can be costly.

> With the free plan the online services are limited. Limited services rely on servers that can be costly.

Author: ```suggestion
With the free plan, the online services are limited. These services rely on servers which can be costly.
```

> If your uses of GDevelop start require more exports each day or a higher number of leaderboards, or you may want to customise the leaderboards of your games you may want to upgrade your free plan to a better plan.

Author: ```suggestion
If you find yourself needing more exports each day or a higher number of leaderboards, or if you want to customize the leaderboards in your games you may want to upgrade your free plan to a better one.
```

See the [pricing page](https://gdevelop.io/pricing) to know more about the advantages of the Silver, Gold, School, and Business subscriptions compared to the free plan.

!!! note

>         ✨ By taking a subscription you helps to the future development of GDevelop. Thank you for your supports!

Author: ```suggestion
        ✨ By taking a subscription you help the future development of GDevelop. Thank you for your support!
```

Once you're logged in, go to your profile as described earlier in the page.
Scroll down, and click on the **Choose a subscription** button beneath **My online services subscriptions**.

![](/gdevelop5/interface/profile/my profile.png)

You will be able to see a description of each available plan. Choose the one that suits your needs by clicking on **Choose this plan**.
Once you've clicked on a plan, you'll be asked for your credit card number so that the subscription can be paid each month.

!!! note

>         Payments are secured, transacted and encrypted using [Stripe.com](https://stripe.com) and [Paypal](https://www.paypal.com/), which are established online payment services. GDevelop do not keept any bank information. Every transactions are secured and handled by the Stripe or Paypal infrastructure.

Reviewer: Payments are secured, transacted, and encrypted using [Stripe.com](https://stripe.com) and [Paypal](https://www.paypal.com/), which are established online payment services. GDevelop does not keep any bank information. Every transaction is secured and handled by the Stripe or Paypal infrastructure.

>         Payments are secured, transacted and encrypted using [Stripe.com](https://stripe.com) and [Paypal](https://www.paypal.com/), which are established online payment services. GDevelop do not keept any bank information. Every transactions are secured and handled by the Stripe or Paypal infrastructure.

Author: ```suggestion
        Payments are secured, transacted, and encrypted using [Stripe.com](https://stripe.com) and [Paypal](https://www.paypal.com/), which are established online payment services. GDevelop does not keep any bank information. Every transaction is secured and handled by the Stripe or Paypal infrastructure.
```

### Changing your subscription

> If you want to update your subscription, either to upgrade to a subscription with more packagings or to a subscription which is cheaper, just open again your profile and click on **Manage subscription** then choose another plan.

Author: ```suggestion
If you want to update your subscription, either to change to a subscription with more flexibility or to a subscription that is cheaper, just open your profile again and click on Manage subscription then choose another plan.
```

> The next payment will automatically be adapted to the new subscription you choosen!

Author: ```suggestion
The next payment will automatically be adapted to the new subscription you have chosen!
```

### Cancelling your subscription

Cancelling your subscription at any time is easy.

 1. Open your profile.
 2. Click on **Manage subscription**.

>  3. See your current subscription, then click on **Cancel Your Subscription** will be immediatelly stopped, no new payment will be required, and all payment stops.

Reviewer: 3. See your current subscription, then click on **Cancel Your Subscription**, no new payment will be required, and all payments will be stopped.

>  3. See your current subscription, then click on **Cancel Your Subscription** will be immediatelly stopped, no new payment will be required, and all payment stops.

Author: ```suggestion
 3. See your current subscription, then click on Cancel Your Subscription, no new payment will be required, and all payments will be stopped.
```

![](/gdevelop5/interface/cancel your subscription.png)

!!! note
		
		If you want, you can later subscribe again to another subscription. In any case, you won't lose any of your games or work!

### Redeem a code

!!! warning

    Please only redeem one subscription at a time, as redeeming multiple subscriptions at once will cancel the previous subscription.
    We recommend you wait for the present subscription to end before activating another.

If you don’t have a GDevelop account yet, create one (you just need an email address, a username, and a password).

Once your account is created, open your profile, click on **Manage subscription** then on **Redeem a code** .
If you’re lost, click on the link below to open your subscription panel in your browser:

[https://editor.gdevelop.io/?initial-dialog=subscription](https://editor.gdevelop.io/?initial-dialog=subscription)

> ![](/gdevelop5/interface/profile/redeem a code.jpg)

Reviewer 2: This image's path is `/gdevelop5/interface/redeem a code.jpg`. You can move the image under the profile folder though

    
Enter your code and press **Redeem**. It’s then applied in a few seconds to your account.

> ![](/gdevelop5/interface/profile/confirming your subscription.png)

Reviewer 2: Same here

That’s it, the confirmation of your account will be shown. You’re ready to create incredible games!

## Contributions

> Here are listed your contribution to the extensions, examples, and assets that are available in the application. Each contribution are public and visible to the public through GDevelop by click on a chip username.

Reviewer: Here is a list of contributions you've made with extensions, examples, and assets that are available in the application. Each contribution is public and visible to the public through GDevelop by clicking on your username.

> Here are listed your contribution to the extensions, examples, and assets that are available in the application. Each contribution are public and visible to the public through GDevelop by click on a chip username.

Author: ```suggestion
Here is a list of contributions you've made with extensions, examples, and assets that are available in the application. Each contribution is public and visible to the public through GDevelop by clicking on your username.
```
